File: pinyougou-parent/pinyougou-manager-web/src/main/webapp/js/service/brandService.js
		//品牌服务层
    	 app.service('brandService',function($http){
    		 //查询所有
    		 this.findAll=function(){
    			 return $http.get('../brand/findAll.do');
    		 }
    		 //分页
    		 this.findPage=function(page,size){
    			 return $http.get('../brand/findPage.do?page='+page +'&size='+size);
    		 }
    		 //加载
    		 this.findOne=function(id){
    			 return $http.get('../brand/findOne.do?id='+id);
    		 }
    		 //添加
    		 this.add=function(entity){
    			 return $http.post('../brand/add.do',entity);
    		 }
    		 //修改
    		 this.update=function(entity){
    			 return $http.post('../brand/update.do',entity);
    		 }
    		 //删除
    		 this.dele=function(ids){
    			return 	$http.get('../brand/delete.do?ids='+ids);
    		 }
    		 //组合条件查询
    		 this.search=function(page,size,searchEntity){
    			 return $http.post('../brand/search.do?page='+page +'&size='+size,searchEntity);
    		 }
    		 //加载品牌下拉列表
    		 this.selectOptionList=function(){
    			 return $http.get('../brand/selectOptionList.do');
    		 }
    	 });

File: pinyougou-parent/pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import java.util.List;
import java.util.Map;

import com.pinyougou.pojo.TbBrand;

import entity.PageResult;



/**
 * 品牌接口
 * @author Administrator
 *
 */
public interface BrandService {
	
	//查询所有品牌
	public List<TbBrand> findAll();
	
	//品牌分页查询
	public PageResult findPage(int pageNum,int pageSize); 
	
	//品牌增加
	public void add(TbBrand brand);
	
	//加载品牌
	public TbBrand findOne(Long id);
	
	//修改品牌
	public void update(TbBrand brand);
	
	//批量删除品牌
	public void delete(Long[] ids);
	
	//品牌分页组合条件查询
	public PageResult findPage(TbBrand brand,int pageNum,int pageSize);
	
	//返回下拉列表数据
	public  List<Map> selectOptionList();
}
